### PR TITLE
Add PyTorch 1.6 to CI, fix errors related to new release

### PR DESCRIPTION
**Pull request by seemethere**
Results from the integration tests that were ran can be found here: https://app.circleci.com/pipelines/github/pytorch/pytorch-integration-testing/28/workflows/80c9c991-949b-4928-a89b-a26c5f023f8f/jobs/149

Ran into an error message when running tests that use this function for
the PyTorch 1.6 release:

> Integer division of tensors using div or / is no longer supported

Replaced the regular divide with a floor divide

### CODE REVIEW

**Comment by ppwwyyxx** [comment]
Thanks @seemethere! In fact we expect to do true division there. When the deprecation happened, Mike already helped us migrate this part of code in D21997444. Not sure why it still needs to be changed?

File: docker/Dockerfile-circleci [hunk 1]
```diff
@@ -14,4 +14,4 @@ RUN wget -q https://bootstrap.pypa.io/get-pip.py && \
 RUN pip install tensorboard opencv-python-headless
 ARG PYTORCH_VERSION
 ARG TORCHVISION_VERSION
-RUN pip install torch==${PYTORCH_VERSION} torchvision==${TORCHVISION_VERSION} -f https://download.pytorch.org/whl/cu101/torch_stable.html
+RUN pip install torch==${PYTORCH_VERSION} torchvision==${TORCHVISION_VERSION} -f https://download.pytorch.org/whl/cu101/torch_stable.html -f https://download.pytorch.org/whl/test/cu101/torch_test.html
```
Comment (ppwwyyxx) [inline docker/Dockerfile-circleci:17]: after stable is released, will this command install 1.6 stable or still 1.6 testing?

Comment (seemethere) [reply docker/Dockerfile-circleci:17]: I believe this should prioritize installs in order of when the indices are noted, for example it should pick out the 1.6 version (if available) in `torch_stable.html` before it picks out the version in `torch_test.html`

**Comment by seemethere** [comment]
> Thanks @seemethere! In fact we expect to do true division there. When the deprecation happened, Mike already helped us migrate this part of code in [D21997444](https://our.intern.facebook.com/intern/diff/D21997444/). Not sure why it still needs to be changed?

I believe the operator alone is not sufficient, and you might need to be more specific given the error message I observed in my testing.

```
>       scale_x, scale_y = (output_width / results.image_size[1], output_height / results.image_size[0])
E       RuntimeError: Integer division of tensors using div or / is no longer supported, and in a future release div will perform true division as in Python 3. Use true_divide or floor_divide (// in Python) instead.
```

cc @mruberry

**Comment by mruberry** [comment]
> > Thanks @seemethere! In fact we expect to do true division there. When the deprecation happened, Mike already helped us migrate this part of code in [[[[[[[[[[[[[[D21997444](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/)](https://our.intern.facebook.com/intern/diff/D21997444/). Not sure why it still needs to be changed?
> 
> I believe the operator alone is not sufficient, and you might need to be more specific given the error message I observed in my testing.
> 
> ```
> >       scale_x, scale_y = (output_width / results.image_size[1], output_height / results.image_size[0])
> E       RuntimeError: Integer division of tensors using div or / is no longer supported, and in a future release div will perform true division as in Python 3. Use true_divide or floor_divide (// in Python) instead.
> ```
> 
> cc @mruberry
> 
> edit: I've updated my PR to use the `true_divide` function

This might happen if output_width and/or output_height are Python integers, which is not a case we encountered previously. New conditionals can be added to test for this and convert the scalars, if needed, or `torch.true_divide` can replace the division, instead.

**Comment by ppwwyyxx** [comment]
Just noticed that when I run the test myself locally with 
```
python3 -m pytest -n 4 -v ./tests/test_model_analysis.py 
```
using pytorch from `pip install torch==1.6.0  -f https://download.pytorch.org/whl/test/cu101/torch_test.html  ` installed just now, the tests are actually passing and I did not observe the error in https://app.circleci.com/pipelines/github/pytorch/pytorch-integration-testing/28/workflows/80c9c991-949b-4928-a89b-a26c5f023f8f/jobs/149. Not sure what is causing the discrepancy.

**Comment by ppwwyyxx** [comment]
We've added 1.6 to CI and tests are passing.

**Pull request closed without merging**